sklearn/metrics/_ranking.py

```diff
@@ -977,6 +985,16 @@ def precision_recall_curve(
     array([1. , 1. , 0.5, 0.5, 0. ])
     >>> thresholds
     array([0.1 , 0.35, 0.4 , 0.8 ])
+    >>>
+    >>> # negative class (y_true = 1) is positively correlated with y_score and positive
+    >>> # class is negatively correlated with y_score
+    >>> import numpy as np
+    >>> from sklearn.metrics import precision_recall_curve
+    >>> y_true = np.array([0, 0, 1, 1])
+    >>> y_scores = np.array([0.1, 0.4, 0.35, 0.8])
+    >>> y_scores_adj = y_scores * (-1)
+    >>> precision, recall, thresholds = precision_recall_curve(
+    ...     y_true, y_scores_adj)
```

Comment:
I am wondering if it would be worthwhile adding an [example](https://scikit-learn.org/stable/auto_examples/index.html) about how to deal with predicted probabilities *and* decision function values that are negatively correlated (instead of adding it here). We would then add a link to this example in the main docstring.

I think it would be nice to spell everything out more completely, which I don't think you can do in these example sections as they are meant to be succinct. 

WDYT @glemaitre ?

Reply:
I let this one open for now. But I'm happy to come up with an example. Can you tell me the exact place to put it?

Reply:
Let's see what @glemaitre says. I am now wondering if explaining it in the user guide would be better...